server/src/main/resources/app.properties

```diff
@@ -8,3 +8,7 @@ prometheus.endpointPortNumber=9999
 # Timeout for consumers to wait for block item before timing out.
 # Default is 1500 milliseconds.
 #consumer.timeoutThresholdMillis=1500
+
+#Persistence Storage Config
+#persistence.storage.rootPath=
+#persistence.storage.type=BLOCK_AS_DIR
```

Comment:
Remember to include newlines at EOF.

Reply:
Hmm should have been done automatically by my IDE, Ill take look, thanks for noticing!

On a side note, is it not possible to ensure this with spotless? Would be good if we can do that.

Reply:
✅ Done.

Reply:
Spotless lacks any significant configuration for the Palantir module; it's an as-is setup.